### PR TITLE
merging chef guide with chef integration page

### PR DESCRIPTION
**Pull request by l0k0ms**
In order to get rid of all guides, We can merge the chef guide into the chef integration page. 
Adding an alias for retro-compatibility.

Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/chef-guide-deletion/integrations/chef/

### CODE REVIEW

**Comment by l0k0ms** [comment]
We need to merge info with this desc too: https://github.com/datadog/puppet-datadog-agent

File: content/integrations/puppet.ja.md [hunk 1]
```diff
@@ -29,4 +29,4 @@ To install the Datadog Agent via Puppet, please see [here](https://github.com/da
 - 管理しているサーバー上のPuppetのパフォーマンスメトリクスを取得する
 - 期待通りに完了しなかったPuppetの実行を検知し、チームと検討する
 
-Puppetを使ってDatadog Agentをインストールする方法に関しては、[Githubの"DataDog/puppet-datadog-agent"](http://docs.datadoghq.com/ja/guides/chef/)を参照してください。
+Puppetを使ってDatadog Agentをインストールする方法に関しては、[Githubの"DataDog/puppet-datadog-agent"](https://github.com/DataDog/puppet-datadog-agent)を参照してください。
```
Comment (irabinovitch) [inline content/integrations/puppet.ja.md]: is this related?

Comment (l0k0ms) [reply content/integrations/puppet.ja.md]: 👍

File: content/integrations/chef.md [hunk 2]
```diff
@@ -5,23 +5,73 @@ kind: integration
...
 
 {{< img src="integrations/chef/chefdashboard.png" alt="Chef Event" >}}
 
 ## Overview
 
-Chef is a popular configuration management tool written in Ruby and Erlang. If you manage your compute instances with Chef and want to use it to install the Datadog agent, [check out our guide for that](http://docs.datadoghq.com/guides/chef/). *This* page is about sending Chef metrics to Datadog.
+Chef is a popular configuration management tool written in Ruby and Erlang. 
 
-Connect Chef to Datadog in order to:
+Deploying Datadog with Chef is meant to be very simple, and provide you with a method of getting the value of monitoring across all of your infrastructure as simply as possible.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: Is it simple or meant to be simple? :)

Comment (l0k0ms) [reply content/integrations/chef.md]: it is :p

File: content/integrations/chef.md [hunk 3]
```diff
@@ -5,23 +5,73 @@ kind: integration
...
+
+Then, on the same page, you must create an Application Key for use with Chef. You may name the key whatever you wish, we recommend something like 'chef_appkey' or something of that nature.
+
+Provide both values in the attributes as shown above.
+
+Then upload your role file to Chef Server like so:
+
+    knife role from file roles/base.rb
+
+The next time Chef runs, it should install the Agent and set the configuration file with the API and application keys.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: "should install"  -> "it will install"

File: content/integrations/chef.md [hunk 4]
```diff
@@ -5,23 +5,73 @@ kind: integration
...
+
+**NOTE:** If you are using another cookbook to define these attributes, use a higher attribute precedence level than `default`.
+
+### Report Handler
+
+Datadog offers a Chef Report Handler which reports metrics and events from your Chef runs to Datadog. Once installed the report handler will submit metrics on Chef run timing and resource changes. Events are also created to allow tracking of Chef run success and failure rates.
+
+This has the added value of bringing the output of a Chef run back to Datadog's Event stream, so failures can be highlighted quickly, discussed amongst the team, and resolved.
+
+Successes typically will be found in the "Low" priority, whereas failures are of "Normal" priority, and when the same node passes the Chef run, then it is put pack into "Low" priority.
```
Comment (irabinovitch) [inline content/integrations/chef.md]: Success will be found in the "Low" priority, whereas failures are of "Normal" priority, and when the same node passes the Chef run, then it is put pack into "Low" priority.

File: content/integrations/chef.md (hunk 3 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/chef.md (hunk 4 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍
